tests/vm_configuration_test.go

```diff
@@ -75,4 +75,75 @@ var _ = Describe("Configurations", func() {
 			Expect(err).ToNot(HaveOccurred())
 		}, 300)
 	})
+
+	Context("New VM with explicitly set VirtIO drives", func() {
```

Comment:
Could you attach a few more disks to the one vm with different bus types and check the result? One or two tests which check multiple buses should be good enough (good for the overall test time since we don't have to boot the vm that often).


Something like:

```yaml
apiVersion: kubevirt.io/v1alpha1
kind: VirtualMachine
metadata:
  name: testvm-ephemeral
spec:
  domain:
    resources:
      requests:
        memory: 64M
    devices:
      disks:
      - name: disk1
        volumeName: disk1
        disk:
          bus: ide
      - name: disk2
        volumeName: disk2
        disk:
          bus: sata
      - name: disk3
        volumeName: disk3
        cdrom:
          bus: virtio
      - name: disk4
        volumeName: disk4
        floppy: {}
      - name: disk5
        volumeName: disk5
        disk:
          bus: virtio
  volumes:
    - name: disk1
      registryDisk:
        image: kubevirt/cirros-registry-disk-demo:devel
    - name: disk2
      registryDisk:
        image: kubevirt/cirros-registry-disk-demo:devel
    - name: disk3
      registryDisk:
        image: kubevirt/cirros-registry-disk-demo:devel
    - name: disk4
      registryDisk:
        image: kubevirt/cirros-registry-disk-demo:devel
    - name: disk5
      registryDisk:
        image: kubevirt/cirros-registry-disk-demo:devel
```

Reply:
Of course, adding such test.